frontend/src/routes/admin/submissions/index.lazy.tsx

```diff
@@ -0,0 +1,255 @@
+import { Badge } from "@/components/ui/badge"
+import { Button } from "@/components/ui/button"
+import {
+  Dialog,
+  DialogContent,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@/components/ui/dialog"
+import { ScrollArea } from "@/components/ui/scroll-area"
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table"
+import { $api } from "@/lib/api"
+import { cn } from "@/lib/utils"
+import { createLazyFileRoute } from "@tanstack/react-router"
+import {
+  createColumnHelper,
+  flexRender,
+  getCoreRowModel,
+  getPaginationRowModel,
+  getSortedRowModel,
+  useReactTable,
+} from "@tanstack/react-table"
+import { ChevronDown, ChevronUp } from "lucide-react"
+import { components } from "openapi/schema"
+import { useMemo, useState } from "react"
+
+type Submission = components["schemas"]["Submission"]
+
+const columnHelper = createColumnHelper<Submission>()
+
+export default function SubmissionList() {
```

Comment:
このリポジトリでは関数は基本アロー関数で書いているので、それに合わせてもらえると助かります！

```suggestion
const SubmissionList = () => {
```